contributing/write-general-writing-hints.md

```diff
@@ -14,63 +14,58 @@ Don't use complicated sentences. Not everyone reading the documentation will be
 
 ### Simple from technical standpoint
 
-Keep the doc simple from a '''technical standpoint''' too. Not everyone reading the documentation will be an experienced admin/user.
+Keep the doc simple from a **technical standpoint** too. Not everyone reading the documentation will be an experienced admin/user.
 
 ## Avoid shorthands
 
 They're just too informal.
 
 ### Bad example:
 
-```
-The update won't be available...
-```
+> The update won't be available...
 
 ### Good example:
 
-```
-The update will not be available...
-```
+> The update will not be available...
 
 ## Be direct
 
 Avoid passive voice.
 
 ### Bad example:
 
-```
-This can be fixed by...
-```
+> This can be fixed by...
 
 ### Good example:
 
-```
-You can fix this issue by...
-```
+> You can fix this issue by...
 
 ## Tell the user what is going on
 
 When you tell users to run a set of commands, tell them what is going on. This mostly applies to cases where the documentation text contains some procedure (a set of commands) to do some complicated stuff. When that happens, don't just put in the commands, but explain them.
 
 ### Bad example:
 
-```
-$ sudo systemctl disable firewalld
-$ sudo systemctl stop firewalld
-$ sudo cp PREUPGRADE_DIR/cleanconf/etc/sysconfig/iptables /etc/sysconfig/iptables
-etc.
-```
+> ```
+> $ sudo systemctl disable firewalld
+> $ sudo systemctl stop firewalld
+> $ sudo cp PREUPGRADE_DIR/cleanconf/etc/sysconfig/iptables /etc/sysconfig/iptables
```

Comment:
Yeah, I definitely understand the concern... my thinking is that the way the examples are formatted now, where **non**-code example text is wrapped in fenced code blocks, is kind of an abuse of &lt;code&gt;. I don't know that I particularly feel that Markdown's "quoted text" block is the _right_ formatting for the examples (I don't like that it renders in a lighter font color), but I'm strongly of the opinion that code blocks _aren't_. Code should be formatted as code. _Prose_ should not. That's how I see it.

One alternative might be to wrap the Bad-vs-Good examples in a table or div, instead of a quoted block, with formatting that helps indicate the contrast between them. (Like, maybe even a pale red background behind the "bad" examples, pale green behind the "good", as an added cue. In the past it might've even been "red font for bad, green font for good", but that tends to be frowned upon these days as it can cause problems for some people with color-deficient vision.) ...I don't know. It's something I can play around with a bit more, if there's any interest. 

I suppose the bad & good examples could all be formatted as Markdown _source_, in which case &lt;code&gt; would be appropriate. But then they should really show the commensurate markup as well, or they're not really "examples". That kind of seems silly, though, since this document isn't discussing _formatting_ style, but rather _writing_ style; the formatting shouldn't be the focus (though it should still be correct, IMHO).